pkgs/sdk/server-ai/README.md

```diff
@@ -0,0 +1,66 @@
+# LaunchDarkly AI SDK (server-side) for .NET
+
+The LaunchDarkly AI SDK (server-side) for .NET is designed primarily for use in multi-user systems such as web servers and applications. It follows the server-side LaunchDarkly model for multi-user contexts. It is not intended for use in desktop and embedded systems applications.
+
+Currently there is no client-side AI SDK for .NET. If you're interested, please let us know by filing an issue!
+
+## LaunchDarkly overview
+
+[LaunchDarkly](https://www.launchdarkly.com) is a feature management platform that serves trillions of feature flags daily to help teams build better software, faster. [Get started](https://docs.launchdarkly.com/home/getting-started) using LaunchDarkly today!
+ 
+[![Twitter Follow](https://img.shields.io/twitter/follow/launchdarkly.svg?style=social&label=Follow&maxAge=2592000)](https://twitter.com/intent/follow?screen_name=launchdarkly)
+
+## Supported .NET versions
+
+This version of the AI SDK is built for the following targets:
+
+* .NET 7.0: runs on .NET 7.0 and above (including higher major versions).
```

Comment:
TODO: I wrote .NET 7.0, but this should actually be 6.0 for the immediate future like the server